js/compileShell.py

```diff
@@ -578,6 +578,7 @@ def fixateVer(pcfile):
         """Returns the current version number (47.0a2)."""
         with open(pcfile, 'rb') as f:
             for line in f:
+                line = line.decode("utf-8", "replace")
```

Comment:
Oh wait, you're right. With encoding specified, it should not be in bytes mode, both in [Python 2](https://docs.python.org/2/library/io.html#io.open) and 3:

```
Traceback (most recent call last):
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 743, in <module>
    main()
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 738, in main
    obtainShell(shell, updateToRev=options.revision)
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 663, in obtainShell
    cfgJsCompile(shell)
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 236, in cfgJsCompile
    compileJs(shell)
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 499, in compileJs
    version = extractVersions(shell.getJsObjdir())
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 592, in extractVersions
    return fixateVer(jspcNewFilename)
  File "/home/gkwubu/funfuzz/js/compileShell.py", line 583, in fixateVer
    with io.open(pcfile, mode='rb', encoding="utf-8", errors="replace") as f:
ValueError: binary mode doesn't take an encoding argument
```

Fixed in rev ec986974eed7110d3ca64624dd62e73c7746f5e4.